src/webview/templates/components/nixpkgs_package.html

```diff
@@ -1,43 +1,45 @@
 {% load viewutils %}
 
-<article class="nixpkgs-package">
-  <h3><span class="pkgs">pkgs.</span>{{ attribute_name }}</h3>
-  <div class="package-meta">
-    <div class="package-description">
-      {{ pdata.description }}
-    </div>
-    <ul class="channel-list">
-      {% for major_channel, major_version in pdata.versions %}
-        <li>
-          <details {% if major_version.uniform_versions %}open{% endif %}>
-            <summary>
-              <span class="branch-major">
-                <span class="branch-name">{{ major_channel }}</span>
-                <a class="version {{ major_version.status }}" target="_blank" href="{{ major_version.src_position }}">
-                  {% if major_version.major_version %}
-                    {{ major_version.major_version }}
-                  {% else %}
-                    ???
-                  {% endif %}
-                </a>
-              </span>
-            </summary>
-            <ul class="channel-list">
-            {% for branch_name, minor_version in major_version.sub_branches %}
-              <li class="branch-minor">
-                <span class="branch-name">{{ branch_name }}</span>
-                <a
-                  class="version {{ minor_version.status }}"
-                  target="_blank"
-                  href="{{ minor_version.src_position }}">
-                    {{ minor_version.version }}
-                </a>
-              </li>
-            {% endfor %}
-            </ul>
-          </details>
-        </li>
-      {% endfor %}
-    </ul>
+<article class="row gap-small spread full-width wrap">
+  <div class="column basis-20 grow">
+    <h3 class="bold"><span class="dimmed">pkgs.</span>{{ attribute_name }}</h3>
+    <p>{{ pdata.description }}</p>
   </div>
+  <ul class="w-20 column">
+    {% for major_channel, major_version in pdata.versions %}
+      <li>
+        <details {% if major_version.uniform_versions %}open{% endif %}>
```

Comment:
Something broke the alignment here

<img width="266" height="51" alt="image" src="https://github.com/user-attachments/assets/24a28842-1727-4cd4-b208-635a08a884a8" />

The monospace font already made a lot of sense for readability.

Reply:
The monospace font isn't what guaranteed alignment before. I tried with monospace vs without and noticed there wasn't a great gain in readability.

Regarding the alignment, titles are aligned, items inside the unfolded content are aligned together. This was intentional to distinguish the inside from the outside better. Though I agree being shifted to the left isn't very natural.

I'll explore ways to improve all this as it is not ideal yet.